spec/API_specification/indexing.rst

```diff
@@ -158,6 +158,11 @@ Multi-dimensional arrays must extend the concept of single-axis indexing to mult
 
     To perform flat indexing, use ``reshape(x, (-1,))[integer]``.
 
+- Each ``None`` in the selection tuple must expand the dimensions of the resulting selection by one dimension of size ``1``. The position of the added dimension must be the same as the position of ``None`` in the selection tuple.
```

Comment:
Do we need to clarify that None is not a "single-axis" index for the purposes of the other rule (like the one below this one)?

Reply:
I've updated the guidance for the `IndexError` rule. This should now be resolved.